### PR TITLE
Updated tox.ini to execute the pep8 tests

### LINKED ISSUE #236: Update tox.ini to execute the pep8 tests
Currently if i do tox -e pep8 it does not run and fails to create the test env. Need to update the tox.ini and add test-requirements for flake8

### PR DESCRIPTION
**Pull request by coolsvap**
Update tox.ini to execute the pep8 tests

Currently if i do tox -e pep8
- it does not run
- fails to create the test env

Updated following,
- tox.ini to include testenv

Added test-requirements for flake8

Resolves #236

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+
+[testenv:pep8]
+commands =
+    flake8
 
 [flake8]
 ignore = E226,E302,E41,E251,C901
 max-line-length = 160
```
Comment (navidshaikh) [inline tox.ini]: I think the max-line-length should be a bit smaller, 100 ?

@goern @vpavlin : thoughts?

**Comment by goern** [comment]
thanks for that. do any requirements for our developers result from your change? (i dont think so but unsure)

**Comment by coolsvap** [comment]
Couple of requirements that will be addressed are

- Check code check conformance on dev env instead of submitting to CI multiple times
- We can always expand on the testenvs as per updates to ci

**Comment by goern** [comment]
great, could you check how that releates to https://github.com/projectatomic/atomicapp/blob/master/CONTRIBUTING.md ? thx!

**Comment by coolsvap** [comment]
@goern please have a look at the updated contributing.md

**Comment by goern** [comment]
LGTM @vpavlin @aweiteka

**Comment by vpavlin** [comment]
LGTM

**Pull request merged**